packages/acceptit-Core.package/ACUserStory.class/class/scenarioAndMethodDict.st

```diff
@@ -1,10 +0,0 @@
-accessing method dictionary
-scenarioAndMethodDict
-
-	"1 halt."
-	^ self methodDict inject: MethodDictionary new into: [:md :cMeth | 
-		((cMeth respondsTo: #pragmaAt:) 
-			ifFalse: [nil] "protect from unnormal cirumstances"
-			ifTrue: [cMeth pragmaAt: #scenario])
-				ifNil: ["normal method" md at: cMeth selector put: cMeth; yourself]
-				ifNotNil: [md at: cMeth getSource asString lines first asSymbol put: cMeth; yourself]].
```

Comment:
@MarcelSchmidberger The magic is happening here: scenario messages were being treated differently. The messagedictionary was being initialized with the sourcestring as a Symbol instead of the methodSelector. I changed this line originally but then realized that it now just does the exact same thing as is being done fore normal methods. so we didn't need the getter for `scenarioAndMethodDict` anymore instead we could just use the normal `methodDict` now all the other methods are overriden methods that normally use the `methodDict` but were being altered to use the `scenarioAndMethodDict` by removing that method overriding wasn't necessary anymore so I just removed those messages aswell.